### PR TITLE
Remove DB error details from the RPC handler response

### PR DESCRIPTION
**Pull request by begmaroman**

### CODE REVIEW

**Review by rachit77 — APPROVED** [review]

File: rpc/rpc.go [hunk 1]
```diff
@@ -65,35 +65,38 @@ func (i *InteropEndpoints) SendTx(signedTx tx.SignedTx) (interface{}, jRPC.Error
...
-	if err := i.executor.Execute(ctx, signedTx); err != nil {
+	if err = i.executor.Execute(ctx, signedTx); err != nil {
 		return "0x0", jRPC.NewRPCError(jRPC.DefaultErrorCode, fmt.Sprintf("failed to execute tx: %s", err))
 	}
 
 	// Send L1 tx
 	dbTx, err := i.db.BeginStateTransaction(ctx)
 	if err != nil {
-		return "0x0", jRPC.NewRPCError(jRPC.DefaultErrorCode, fmt.Sprintf("failed to begin dbTx, error: %s", err))
+		log.Errorf("failed to begin dbTx, error: %s", err)
```
Comment (vcastellm) [inline rpc/rpc.go:84]: every time it returns here, will log an error, so this is redundant?

File: rpc/rpc.go [hunk 2]
```diff
@@ -65,35 +65,38 @@ func (i *InteropEndpoints) SendTx(signedTx tx.SignedTx) (interface{}, jRPC.Error
...
 		if errRollback := dbTx.Rollback(ctx); errRollback != nil {
 			log.Error("rollback err: ", errRollback)
 		}
 		return "0x0", jRPC.NewRPCError(jRPC.DefaultErrorCode, fmt.Sprintf("failed to add tx to ethTxMan, error: %s", err))
 	}
-	if err := dbTx.Commit(ctx); err != nil {
-		return "0x0", jRPC.NewRPCError(jRPC.DefaultErrorCode, fmt.Sprintf("failed to commit dbTx, error: %s", err))
+
+	if err = dbTx.Commit(ctx); err != nil {
+		log.Errorf("failed to commit dbTx, error: %s", err)
```
Comment (vcastellm) [inline rpc/rpc.go:96]: every time it returns here, will log an error, so this is redundant?

File: rpc/rpc.go (hunk 1 above)
Comment (begmaroman) [reply rpc/rpc.go:84]: Not necessarily, it returns a short error without details but logs the full error message. It was the main idea of this PR.

**Review by vcastellm — APPROVED** [review]
LGTM

**Pull request merged**